tests/wicked/basic/ref/t08_setup_second_card.pm

```diff
@@ -16,11 +16,13 @@ use base 'wickedbase';
 use strict;
 use warnings;
 use testapi;
-
+use utils qw(systemctl);
 
 sub run {
     my ($self, $ctx) = @_;
     record_info('Info', 'Set up a second card');
+    $self->get_from_data('wicked/dhcp/dhcpd_2nics.conf', '/etc/dhcpd.conf');
+    systemctl 'restart dhcpd.service';
```

Comment:
I just remember: https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/8353e79ef7c1c5dd1c70c4491a4a90f1c403e8ce#diff-c427fe73165599fbc5799ecdc24b7a01R28

Reply:
done

Reply:
> I just remember: [8353e79#diff-c427fe73165599fbc5799ecdc24b7a01R28](https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/8353e79ef7c1c5dd1c70c4491a4a90f1c403e8ce#diff-c427fe73165599fbc5799ecdc24b7a01R28)

why should mutex create fail in that case? I am ok to change it but not sure if needed.

Reply:
+1 to fail when mutex not created . You creating it with a good reason to make sure that two test instances doing actions in right time spot. In case mutex is not created test most probably fail to but - there is big chance that failure would be not so obvious and there is a big chance that you will waste 2 hours of worker time because it will wait for mutex which failed to be created

Reply:
done

Reply:
Regarding that 2h waiting problem. I was thinking in adding a timeout parameter to mutex_wait(). I think in most of the cases it would help. Just for the next  PR :)